discord/help.py

```diff
@@ -13,10 +13,11 @@ def __init__(self, bot):
         self.bash = "<:bash:831116845133332500>"
         self.js = "<:nodejs:834358450309300265>"
         self.cpp = "<:cpp:917582153544507442>"
+        self.haskell = "<:haskell:310831928360304653>"
```

Comment:
Will you please send me this emoji so I can put it in my private server with the other ones?

Reply:
Yeah, I'll send it to you on Discord.